### DISCUSSION TITLE
[question] How can I query the version of lxdware ?

### DISCUSSION THREAD

**Discussion by erik78se** [lxdware/lxd-dashboard#47] [Q&A]
Is there a way to query what the version of the current lxdware is installed?

I've been looking all over but there is only a static string in some php file that I can use.

I would need something to be able to check the version as part of some monitoring for a Juju charm I'm putting together with lxdware!

Thanx!

**Comment by matthewalanpenning** [comment]
That is a great question. The file that is updated and display's the version is https://github.com/lxdware/lxd-dashboard/blob/master/lxd-dashboard/backend/config/about.php on line 46.
I just manually change it with a new release. If it helps I could look at creating a php file with a version variable for the next release.

**Comment by erik78se** [comment]
The ability to do a few things from command line would be great!

* Ability to retrieve the server.crt via a command line to automate
integration?
* Get the version number and other release details as a json - so simplify
knowing what is actually running and do a basic health check.
* A means to automate adding and removing hosts.

Thanx alot - I will share my charm code as soon as I can with you!

Den tis 30 aug. 2022 02:54Matthew Penning ***@***.***> skrev:

**Comment by matthewalanpenning** [comment]
Hello Erik,

Currently the only place I have the version number is on the
https://github.com/lxdware/lxd-dashboard/blob/master/lxd-dashboard/backend/config/about.php
file line 46. If it would help I could create a file that would hold the
version number that could easily be parsed. What would you suggest?

Thanks,
Matt
